qiskit/schemas/qobj_schema.json

```diff
@@ -153,6 +154,48 @@
                     ],
                     "title": "Frame change pulse"
                 },
+                {
+                    "properties": {
+                        "name": {
+                            "enum": [
+                                "delay"
+                            ]
+                        },
+                        "duration": {
```

Comment:
If I look at `acquire` below the definition of duration and other parameters is much simpler. Could we use this form?